parent 5f0a4d71a9e0614b9b3a9cf36397f53ed8c7f014 ()
commit ab22843236163656aa602afe0bde9cbc7c405959
Author: Jan Engelhardt <jengelh@medozas.de>
Date:   Sun Aug 16 14:22:57 2009 +0200

cryptsetup: remove conflicting status message

Users of libpam-mount regularly get confused when PAM is set to
no-debug-output (which is the default), but cryptsetup spews this
line. Commands that ran successful should not print to stderr,
because that would indicate a warning, and "command successful"
definitely is not a warning.
---
 src/cryptsetup.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/src/cryptsetup.c b/src/cryptsetup.c
index 075a12a..5ce722f 100644
--- a/src/cryptsetup.c
+++ b/src/cryptsetup.c
@@ -125,10 +125,8 @@ static void show_status(int errcode)
 {
 	char error[256];
 
-	if(!errcode) {
-                fprintf(stderr, _("Command successful.\n"));
+	if(!errcode)
                 return;
-	}
 
 	crypt_get_error(error, sizeof(error));
 	if (!opt_verbose) {
-- 
# Created with git-export-patch
